content/blog/kubecon_eu_2018.md

```diff
@@ -0,0 +1,62 @@
++++
+categories = ["Community"]
+date = "2018-05-11"
+tags = ["events", "kubernetes", "kubecon"]
```

Comment:
You may want to add some more like `"CNCF", "cloud", "community-update" , "events", "presentation"`

Reply:
added additional tags